rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -155,6 +158,7 @@ exit 0
 %exclude %{gem_cache}
 %{gem_spec}
 %{foreman_bundlerd_plugin}
+%config(noreplace) %{foreman_pluginconf_dir}/%{gem_name}.yaml
```

Comment:
Is noreplace deliberate?  (I know we have this in error in some other packages, so just checking.)

noreplace means all updates go to .rpmnew, while %config alone would mean _unmodified_ files will be updated, but user-modified ones are left and the new copy becomes .rpmnew, IIRC.